src/org/labkey/test/components/ui/EntityInsertPanel.java

```diff
@@ -190,8 +204,13 @@ protected ElementCache newElementCache()
 
     protected class ElementCache extends Component.ElementCache
     {
+        Locator deleteRowsBtnLoc = Locator.XPathLocator.union(
+                Locator.button("Delete rows"),
+                Locator.buttonContainingText("Remove"));
+
         WebElement bulkInsert = Locator.button("Bulk Insert").findWhenNeeded(_driver);
-        WebElement deleteRows = Locator.button("Delete rows").findWhenNeeded(_driver);
+        WebElement bulkUpdate = Locator.button("Bulk update").findWhenNeeded(_driver);
+        WebElement deleteRows = deleteRowsBtnLoc.findWhenNeeded(_driver);
```

Comment:
Should the elements in this cache be scoped to the component instead of the entire page? e.g.
```suggestion
        WebElement deleteRows = deleteRowsBtnLoc.findWhenNeeded(this);
```

Reply:
I've scoped these finders down to the insertPanel